starter_project/api/app/constraints_routes.js

```diff
@@ -4,9 +4,8 @@ module.exports = (() => {
     const {getConstraints} = require('./constraints_actions.js');
   
     const getConstraintsRoute = (req, res) => {
-
-      getConstraints().then((constraint) => {
-        console.log('constraints');
+      getConstraints().then((constraints) => {
```

Comment:
Changing constraint to constraint**s** fixed the bug

Reply:
Got it. It's good, green light from me! :+1: